tests/lax_numpy_test.py

```diff
@@ -1658,6 +1658,21 @@ def testPolyMul(self, a1_shape, a2_shape, dtype):
     self._CheckAgainstNumpy(np_fun, jnp_fun_np, args_maker, check_dtypes=False, tol=tol)
     self._CompileAndCheck(jnp_fun_co, args_maker, check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "a_shape={}_trim={}".format(
+          jtu.format_shape_dtype_string(a_shape, dtype), trim),
+       "dtype": dtype, "a_shape": a_shape, "trim":trim}
+      for a_shape in one_dim_array_shapes
+      for dtype in default_dtypes
+      for trim in ['f', 'b', 'fb']))
+  def testTrimZeros(self, a_shape, dtype, trim):
+    rng = jtu.rand_default(self.rng())
+    np_fun = lambda arg1, arg2: np.trim_zeros(arg1, trim=arg2)
+    jnp_fun = lambda arg1, arg2: jnp.trim_zeros(arg1, trim=arg2)
+    args_maker = lambda: [rng(a_shape, dtype), trim]
```

Comment:
Generally our test frameworks uses lambdas to bind static parameters, and args_maker to generate random inputs. So it would look something like this:
```python
    np_fun = lambda filt: np.trim_zeros(filt, trim=trim)
    jnp_fun = lambda filt: jnp.trim_zeros(filt, trim=trim)
    args_maker = lambda: [rng(a_shape, dtype)]
```